### DISCUSSION TITLE
How long will it take to process my request?

### DISCUSSION THREAD

**Discussion by AmirAli-AZ** [community/community#42690] [GitHub Education]
Hi github.
I have sent a request for github student pack 3 days ago
I don't have school email so I provided valid documents.
![SmartSelect_20221227_105646_Chrome](https://user-images.githubusercontent.com/84965706/209628913-66fbb21d-7a94-4906-8969-9a57c919258c.jpg)

**Comment by amalpotra** [comment]
According to this [blog](https://github.blog/2019-07-30-how-to-get-the-github-student-developer-pack-without-a-student-id/), if your school email is trusted, it could take as little as a few minutes. Uploaded proof requires additional scrutiny, which may take more time to review. GitHub does their best to review all applications within two weeks of being submitted. The application process may take a little longer during times where they receive high volumes of applications, such as the start of a new semester.

**Reply by AmirAli-AZ** [reply]
Thank you for your reply

**Reply by Marco9996** [reply]
Ciao

**Comment by jfullstackdev** [comment]
I'm seeing a lot of unaccepted applications here, so there is no assurance, but being complete with all the requirements is a plus

I actually waited for Codespaces for, I think, 6 months, it's worth the wait

**Comment by logtang** [comment]
Very late to this, but how do you see that you have a pending application?
